### PR TITLE
adding digitalocean terraform installation guide

### PR DESCRIPTION
**Pull request by jmarhee**
I based this on the packet.md, since the Terraform templates effectively just extended those for Packet as well. I made provider-specific changes and additions, but nothing inconsistent with the original guides, so hopefully that'll minimize any confusion/style issues.

### CODE REVIEW

**Comment by mhausenblas** [comment]
Thanks a lot @jmarhee! Since you're a TF expert @jgarcia-mesosphere, would you like to review and test-drive it and give us a LGTM?

File: 1.7/administration/installing/cloud/digitalocean.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+# Create a DCOS cluster
+
+## Prerequisites:
+
+- [DigitalOcean API Key](https://www.digitalocean.com/help/api/)
+
+- [Terraform by Hashicorp](https://www.terraform.io/intro/getting-started/install.html)
+
+## Installing DCOS
```
Comment (pyronicide) [inline 1.7/administration/installing/cloud/digitalocean.md]: Don't forget the / - `DC/OS`. Looks like there's a couple of these.

File: 1.7/administration/installing/cloud/digitalocean.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+You may need to wait a few minutes from this point for all the DCOS services to become active and the control panel available on the master node. After 15 or 20 minutes, you'll want to check out the [troubleshooting](../../custom/troubleshooting/) documentation.
+
+# Launch DCOS
+Launch the DCOS web interface by entering the Mesos master IP address:
+
+1.  Cut/paste the link provided by after running terraform apply, or by running terraform output from the same directory, into your browser to open the DCOS web interface. The interface runs on the standard HTTP port 80, so you do not need to specify a port number after the hostname.
+
+2.  Install the DCOS Command-Line Interface (CLI). You can install the CLI to administer your DCOS cluster. You can access the documentation at any time by clicking the caret icon in the lower left corner.
+
+    ![dcos help link](../../../img/packet_help_link.png)
```
Comment (pyronicide) [inline 1.7/administration/installing/cloud/digitalocean.md]: Sure you don't want a DO specific screen?

File: 1.7/administration/installing/cloud/digitalocean.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    $ ssh-keygen -t rsa -f ./do-key
+    ```
+
+4.  Copy `sample.terraform.tfvars` to a new file named `terraform.tfvars`, and edit the new file, filling in the values as desired. The following are blank and if not filled in, you will be prompted by terraform when necessary:
+
+    - digitalocean_token - Your DigitalOcean API key
+    
+    - ssh_key_fingerprint - Your SSH public key, or fingerprint
+
+    - dcos_installer_url - Where to get DCOS
```
Comment (pyronicide) [inline 1.7/administration/installing/cloud/digitalocean.md]: Maybe (default - https://downloads.dcos.io) ?

**Comment by pyronicide** [comment]
How do the firewall rules work? Are all the masters and agents open to the internet?

**Comment by jmarhee** [comment]
I made the recommended changes to the typesetting (DCOS -> DC/OS) that carried over from the Packet template (my apologies; was trying to remain consistent with the existing docs, so I overlooked that).

Added DO-specific images (so the terraform output shows DO IP space, and the help screen shows an @digitalocean.com cluster prefix and user email. 

As far as the firewall goes, I did not do any additional configuration, so yes, they may be Internet facing. This process was not changed from the Terraform configs I forked my repo from (packet-terraform), but I can include that component (because I do recall there being a VPN component with Mesosphere on DO, which I overlooked in doing my write-up on this, so again, my apologies), if desired for installation docs.

**Comment by jgarcia-mesosphere** [comment]
Needless to say, I'll be looking to make similar changes to dcos/packet-terraform.

**Comment by pyronicide** [comment]
@jmarhee a disclaimer that everything is open by default and security is a concern is sufficient for me for now, I'd love to see the cluster get secured by default in the future though.

**Comment by jmarhee** [comment]
@pyronicide Fair enough; I'll add a disclaimer to that effect, and keep security on my roadmap for any improvements to the Terraform script, as this is imminently doable (but admittedly had not been on my mind) on DigitalOcean.

**Comment by jmarhee** [comment]
Will the disclaimer I added suffice, or should I re-commit with something more like @jgarcia-mesosphere's in https://github.com/dcos/packet-terraform/issues/3? I don't want to cause any problems with standardization/phrasing you might prefer be used to describe this particular warning.

**Comment by pyronicide** [comment]
Ahha, I just saw it. I'd like to see something at the top of the post instead of the bottom. It is a pretty big disclaimer and I'd hate for someone to not see it.

**Comment by jmarhee** [comment]
Definitely a good point. I moved it to the top of the document right under the overview, so it precedes any actions or configuration options.

**Comment by pyronicide** [comment]
Looks great!

**Pull request merged**